docs/reference/configuration/keywords.rst

```diff
@@ -1371,22 +1371,22 @@ Keywords controlling the ES algorithm
 .. _enkf_alpha:
 .. topic:: ENKF_ALPHA
 
-        This controls the scaling factor used when detecting outliers. Increasing this
+        The scaling factor used when detecting outliers. Increasing this
         factor means that more observations will potentially be included in the
         assimilation. The default value is 3.00.
 
         Including outliers in the Smoother algorithm can dramatically increase the
         coupling between the ensemble members. It is therefore important to filter out
-        these outlier data prior to data assimilation. An observation, :math:`\textstyle
-        d^o_i`, will be classified as an outlier if
+        these outliers data prior to data assimilation. An observation,
+        :math:`\mathbf{d}^o_i`, will be classified as an outlier if
```

Comment:
To me, a single observation d_i is a value and not a vector. Could you explain a bit more?